graylog-storage-elasticsearch6/src/main/java/org/graylog/storage/elasticsearch6/IndexFieldTypePollerAdapterES6.java

```diff
@@ -51,7 +52,7 @@ public IndexFieldTypePollerAdapterES6(JestClient jestClient) {
         final JsonNode properties = result.getJsonObject()
                 .path(indexName)
                 .path("mappings")
-                .path("message") // TODO: Hardcoded index type name
+                .path(IndexMapping.TYPE_MESSAGE) // TODO: Hardcoded index type name
```

Comment:
Uhm, that comment can go now, as it's no longer hard coded, is it?

Reply:
:heavy_check_mark: